Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/CertificateStateHelper.kt

```diff
@@ -177,7 +177,7 @@ fun PersonOverviewItemBinding.setUIState(
                 thirdCertificate?.cwaCertificate
             )
         }
-        certificateToggleGroup.check(0)
+        certificateToggleGroup.check(firstCertificateButton.id)
```

Comment:
should we change the id based on what is selected ? there could be 3 certificate buttons ,right?

Reply:
This is just for initialisation, the MaterialButtonToggleGroup handles checking the button by itself, so it is not needed to additionally handle checking in click listener

Reply:
👍🏼